homework_1.py

```diff
@@ -0,0 +1,50 @@
+while True:
+    print("Enter command: ")
+    command = str(input())
```

Comment:
`input` уже возвращает строку, преобразование к `str` излишне. Приглашение по вводу можно вывести с использованием `input`
```suggestion
    print("Enter command: ")
    command = str(input())
```
```suggestion
    command = input("Enter command: ")
```